src/plugins/data/public/query/timefilter/get_time.ts

```diff
@@ -19,7 +19,9 @@
 
 import dateMath from '@elastic/datemath';
 import { TimeRange } from 'src/plugins/data/public';
-import { IndexPattern, Field } from '../index_patterns';
+
+// TODO: remove this
+import { IndexPattern, Field } from '../../../../../legacy/core_plugins/data/public/index_patterns';
```

Comment:
This seems like a no-no&mdash;importing from the Legacy Platform in the New Platform. My understanding is that we should not be doing this even with `TODO` comment, but maybe I'm wrong (/cc @ppisljar @joshdover ).

Reply:
Talked about this with @ppisljar already.
We decided to go with it, because otherwise we're blocked until `IndexPatterns` are done. It's a **temporary solution** that works since it's only a type, and once IndexPatterns are migrated, we'll switch it over.

Reply:
its only types that we are importing, so this should be ok